### PR TITLE
Reformatting the files in the scripts folder

### PR DESCRIPTION
**Pull request by ytimocin**
# Description
Reformatting the files in the scripts folder

## Type of change
- This pull request is a minor refactor, code cleanup, test improvement, or other maintenance task and doesn't change the functionality of Radius (issue link optional).

### CODE REVIEW

File: .github/scripts/get_release_version.py [hunk 1]
```diff
@@ -62,7 +62,8 @@
 
 with open(os.getenv("GITHUB_ENV"), "a") as githubEnv:
     if gitRef is None:
-        print("This is not running in github, GITHUB_REF is null. Assuming a local build...")
+        print(
+            "This is not running in github, GITHUB_REF is null. Assuming a local build...")
```
Comment (lakshmimsft) [inline .github/scripts/get_release_version.py:66]: original code seems more readable and it is not really getting into multi lines

File: .github/scripts/transform_test_results.py [hunk 2]
```diff
@@ -22,9 +22,11 @@
 import sys
 import xml.etree.ElementTree
 
+
 def main():
     if len(sys.argv) != 4:
-        print("Usage: transform-test-results.py <repository root> <input file> <output file>")
+        print(
+            "Usage: transform-test-results.py <repository root> <input file> <output file>")
```
Comment (lakshmimsft) [inline .github/scripts/transform_test_results.py:29]: same question here as above

File: .github/scripts/get_release_version.py (hunk 1 above)
Comment (ytimocin) [reply .github/scripts/get_release_version.py:66]: I think this is how a python file gets reformatted.

Comment (ytimocin) [reply .github/scripts/get_release_version.py:66]: after a certain character count in a given line

**Review by lakshmimsft — APPROVED** [review]

**Pull request merged**